### DISCUSSION TITLE
Definicion del proyecto Imposible.

### DISCUSSION THREAD

**Discussion by Gona79** [ISPC-TST-EM2022/Grupo1#194] [Announcements]
Que tal estimados, les dejo este espacio para que definan su proyecto y lo expliquen brevemente. De esta forma podremos habrir el registro con el profesor Jorge. Haganlo con el siguiente formato: 

Tema del Proyecto: 
Nombre del Proyecto:  
Definicion del Proyecto:

**Comment by FedeBirge** [comment]
Hola Profe, recién vemos el mensaje, nos tomamos el finde sin antes dejar cargado en el repo lo necesario para ésta primer semana, como presentación, diagrama de Gantt, diagrama en bloques, ficha de revisión y seguimiento. Investigación de sensores a usar, y problemática actual.

Tema del Proyecto: Software Libre
Nombre del Proyecto: DIAC (dispositivo inteligente antirrobo de Cubiertas)
Definición del Proyecto: PDF con presentación. En resumen un dispositivo que detecte presencia cuando se intenta sustraer una cubierta, genere una advertencia, seguido de la activación de una alarma, dándole soporte al usuario en su teléfono móvil

**Comment by Gona79** [comment]
**Chicos trabajaron muy bien:** 

- [x] Manejo de Control de versiones: Ok
- [x] Manejo de Estructura de proyecto:  Ok
- [ ] Presentacion Final: Falta desarrollar, la presentacion grupal.

**Reply by FedeBirge** [reply]
Hola profe, gracias por la devolución, entendíamos que esta semana era de presentación, la cual vamos a estar haciendo y subiendo

**Comment by JorEl057** [comment]
Devolución Grupal: @FedeBirge  @LitoCarunchio  @estebancarrizo  @marlufer  @Emygut @GiseRomero 
Nota Final Grupal: 8 (Ocho)

Ideación del Proyecto: 8 (Ocho)
Presentación: 6 (Seis)
Uso de recursos: 10 (Diez)
Desarrollo: 8 (Ocho)
Implementación: 8 (Ocho)

Nota Final Grupal: 8 (Ocho)

Estimados tomaron la idea del Dispositivo Inteligente Antirrobo de Cubiertas (DIAC) como proyecto y lo ampliaron como un prototipo con la posibilidad de ser independiente del sistema de alarma del vehículo, aplicable a cualquier tipo de automóviles, motocicletas y grandes rodados. Esto demuestra su compromiso y RSU. En su presentación que cumplieron correctamente con un tiempo acorde, hablan sobre la necesidad en forma general y la implementación en forma particular. Trabajaron demostrando una organización y de recursos agiles muy bien logrados. Finalmente el desarrollo y la implementación del proyecto también fueron logrados. Felicitaciones y continúen trabajando así!!!
